.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,13 @@
 
 #### How should this be manually tested?
 
+#### Do the tests adhere to the following recommendations?
+
+- [ ] `screen` is used for querying.
+- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
+- [ ] `userEvent` is used in place of `fireEvent`.
+- [ ] `asFragment()` is used for snapshot testing.
```

Comment:
separate from this PR, but wondering if we should convert a test to this so that users unfamiliar could find an example in the repo. Right now, there's nothing using that so it's hard to model after

Reply:
Yeah I agree it would be good to have an example. I'm thinking of opening a separate PR that converts some of the tests for a component to use best practices and this can include an example of the recommended way to do snapshot testing